xchange-gatecoin/.gitignore

```diff
@@ -0,0 +1,27 @@
+# IntelliJ
```

Comment:
This shouldn't be here as far as I understand because of the .gitignore in the parent directly.